spec/spec_helper.rb

```diff
@@ -64,7 +64,7 @@ def committed_at
   end
 
   config.around :each do |example|
-    Async.run do
+    Async do
```

Comment:
Perhaps this would do as we can't run in parallel anyway
```suggestion
    Sync do
```

Reply:
But fine to keep as it was like that

Reply:
I think it's fine to change it. https://github.com/3scale/apisonator/pull/427/commits/a2ed9ae99855f31999323c173af08010f630fb0e